demo/webclient/README.md

```diff
@@ -0,0 +1,81 @@
+
+# WebRTC Web Client
+
+This is a dockerized version of the Mediasoup web client demo (v3) with ICE server handling, specifically modified for the 2060 project. The primary goal is to decouple the web client from the server, as demonstrated in the original Mediasoup demo.
+
+For this version, the `RoomClient` module has been refactored to fetch the Mediasoup server configuration through the `createWebRtcTransport` request. This configuration includes activating ICE mode and specifying the ICE servers for stream transmission and reception.
+
+## Prerequisites
+
+To run this client, you need an online Mediasoup server with ICE parameters configured. Make sure the server's `config.js` file includes the following settings:
```

Comment:
It might be a good practice to mention environment variables when creating the `.env` or `docker-compose-yml` file during the initial configuration.

Reply:
The WebRTC server configurations are defined in its documentation. Let's remember that we are working on the web client, and the reference to this configuration is only to emphasize the functionality of the iceServer feature.

Reply:
Added Job build-and-push-webclient into demos.yml